src/models/lineChart.js

```diff
@@ -225,14 +225,17 @@ nv.models.lineChart = function() {
                     })
                     .forEach(function(series,i) {
                         pointIndex = nv.interactiveBisect(series.values, e.pointXValue, chart.x());
-                        lines.highlightPoint(i, pointIndex, true);
                         var point = series.values[pointIndex];
+                        var pointYValue = chart.y()(point, pointIndex);
+                        if (pointYValue != null) {
+                            lines.highlightPoint(i, pointIndex, true);
+                        }
```

Comment:
Looks like stackedAreaChart.js, lineWithFocusChart.js, and cumulativeLineChart.js are the only other ones with multiple lines that also have the interactive guideline option.  Not sure if this specific problem happens with those though.  Making null be zero is probably valid for stackedAreaChart and cumulativeLineChart, so probably only lineWithFocusChart needs to be fixed in the same way...